### PR TITLE
Allow newest phpcs rules

### PR DESCRIPTION
**Pull request by daniel-rose**

### CODE REVIEW

**Review by julianzimmermann — APPROVED** [review]

File: bundles/jellyfish-credit-memo/src/FondOfOryx/Zed/JellyfishCreditMemo/Business/Api/Adapter/CreditMemoAdapter.php [hunk 1]
```diff
@@ -114,7 +114,7 @@ protected function createOptions(AbstractTransfer $transfer): array
         $options = [];
 
         $options[RequestOptions::HEADERS] = static::DEFAULT_HEADERS;
-        if (!empty($this->username) && !empty($this->password)) {
+        if ($this->username !== '' && $this->password !== '') {
```
Comment (DerCalli) [inline bundles/jellyfish-credit-memo/src/FondOfOryx/Zed/JellyfishCreditMemo/Business/Api/Adapter/CreditMemoAdapter.php:117]: is it correct that both have to be empty?

File: bundles/payment-epc-qr-code/src/FondOfOryx/Service/PaymentEpcQrCode/Model/Builder/EpcDataBuilder.php [hunk 2]
```diff
@@ -30,7 +30,7 @@ public function fromTransfer(PaymentEpcQrCodeRequestTransfer $paymentEpcQrCodeRe
         $reference = $paymentEpcQrCodeRequestTransfer->getReference() ?? '';
         $purpose = $paymentEpcQrCodeRequestTransfer->getPurpose() ?? '';
 
-        if (empty($usage) === false && empty($reference) === false) {
+        if ($usage !== '' && $reference !== '') {
```
Comment (DerCalli) [inline bundles/payment-epc-qr-code/src/FondOfOryx/Service/PaymentEpcQrCode/Model/Builder/EpcDataBuilder.php:33]: Same here, what happend if only one is empty

Comment (julianzimmermann) [reply bundles/payment-epc-qr-code/src/FondOfOryx/Service/PaymentEpcQrCode/Model/Builder/EpcDataBuilder.php:33]: Watch lines below, one has to be empty. It should fail if both are set

File: bundles/jellyfish-credit-memo/src/FondOfOryx/Zed/JellyfishCreditMemo/Business/Api/Adapter/CreditMemoAdapter.php (hunk 1 above)
Comment (julianzimmermann) [reply bundles/jellyfish-credit-memo/src/FondOfOryx/Zed/JellyfishCreditMemo/Business/Api/Adapter/CreditMemoAdapter.php:117]: if one or both are empty, there will be no auth header set

**Review by gengjozsef — APPROVED** [review]

**Pull request merged**